### PR TITLE
fix(server): Add lerc external into server package.

### PR DESCRIPTION
**Pull request by Wentao-Kuang**
### Motivation
It looks like we are missing lerc in the server cli. 
https://github.com/linz/basemaps-config/actions/runs/11061552896/job/30734619452?pr=970
image

### Modifications
Add the `lerc` into server and cli package

### Verification

Local build server container with lerc uninstall vs installed.

#### UNINSTALL
image

#### INSTALLED
image

#### Test Screenshot with fixed container
image

### CODE REVIEW

File: packages/server/Dockerfile [hunk 1]
```diff
@@ -12,6 +12,8 @@ WORKDIR /app/basemaps
 COPY ./basemaps-landing*.tgz /app/basemaps/
 COPY ./basemaps-server*.tgz /app/basemaps/
 
+RUN npm install lerc@^4.0.4
```
Comment (blacha) [inline packages/server/Dockerfile:15]: why is this needed here? if we were missing binary deps then sharp should be missing too?

File: packages/cli/Dockerfile [hunk 2]
```diff
@@ -15,7 +15,7 @@ RUN apt-get update
 RUN apt-get install -y nodejs
 
 # Install sharp TODO update this when we change sharp versions
-RUN npm install sharp@0.33.0
+RUN npm install sharp@0.33.0 lerc@^4.0.4
```
Comment (blacha) [inline packages/cli/Dockerfile:18]: where in the CLI is this needed?

Comment (Wentao-Kuang) [reply packages/cli/Dockerfile:18]: We use this cli container for the server in [basemaps-config](https://github.com/linz/basemaps-config/blob/master/scripts/bms.sh). So we need to install lerc this into cli container.

File: packages/server/Dockerfile (hunk 1 above)
Comment (Wentao-Kuang) [reply packages/server/Dockerfile:15]: I didn't see an error related sharp when I test the server container with screenshots. Might be the deps for `cogify` in the `cli` container? But we definitely need to install `lerc` in here to make it work.

File: packages/cli/Dockerfile (hunk 2 above)
Comment (blacha) [reply packages/cli/Dockerfile:18]: if the server is being installed it should come with `lerc` so I think the server is just missing the lerc dep? how is `sharp` defined in basemaps/server ?

Comment (Wentao-Kuang) [reply packages/cli/Dockerfile:18]: `sharp` is installed as dependency already in the [basemaps/server](https://github.com/linz/basemaps/blob/master/packages/server/package.json#L50)

But we are not installing `sharp` in the basemaps/cli, so we installed into container directly.

Comment (Wentao-Kuang) [reply packages/cli/Dockerfile:18]: I don't think server package is installed in the container, it is just a [function call](https://github.com/linz/basemaps/blob/master/packages/cli/src/cli/server/action.serve.ts) to the `package/server`, so we still need to install `lerc` in the cli container.

File: packages/cli/src/cli/server/action.serve.ts [hunk 3]
```diff
@@ -9,6 +9,7 @@ import {
 
 const DefaultPort = 5000;
 
+//TO DEPRECATE: We should use the server container directly now, and deprecate this later.
```
Comment (blacha) [inline packages/cli/src/cli/server/action.serve.ts:12]:
```suggestion
/** @deprecated 2024-09 Use the basemaps/server container directly now, and deprecate this later. */
```

**Review by blacha — APPROVED** [review]
